material-components/material-components-android-examples
# [Reply] Add Search Bar

# Summary
- Added Searchbar in the form of MaterialToolbar
- Ripple effects, and transition to search screen on click of the search Bar
- Cannot upload a video, gifs are too large and Github doesn't support WEBM

# Test
- Pull Branch
- Make sure searchbar is looking up to par and transitions to the Search Screen on click.

# Screenshots
<img src="https://user-images.githubusercontent.com/32623283/140411589-4b8df633-962d-42e3-9c27-ef0c2d961b1c.png" width="300">

<img src="https://user-images.githubusercontent.com/32623283/140411622-567665c7-4c2f-4988-b18b-498b405b396d.png" width="300">
